pages/en/developer/assemblyscript-migration-guide.mdx

```diff
@@ -171,16 +171,17 @@ Examples:
 
 ```typescript
 // primitive casting
-let a: usize = 10
-let b: isize = 5
-let c: usize = a + (b as usize)
+let a: usize = 10;
+let b: isize = 5;
+let c: usize = a + (b as usize);
 ```
 
 ```typescript
 // upcasting on class inheritance
 class Bytes extends Uint8Array {}
 
-let bytes = new Bytes(2) < Uint8Array > bytes // same as: bytes as Uint8Array
```

Comment:
This is how Prettier will reformat these two lines of code (presumably a bug), so maybe change them to something like:
```
let bytes = new Bytes(2)
// `<Uint8Array>bytes` is equivalent to `bytes as Uint8Array`
```

Reply:
Fixed!